client/src/data/copy/explore.tsx

```diff
@@ -358,14 +358,14 @@ export const DOWNLOAD_DRAFT = {
     values={{
       downloadDraft:
         <a href={METHODOLOGY_COPY.DOWNLOAD_ZIP_URL}>
-          {'Download the draft list'}
+          {`Download the draft list v${METHODOLOGY_COPY.VERSION_NUMBER}`}
```

Comment:
@vim-usds where does this get set?

Reply:
@switzersc-usds , great question. All (if not close to all) of the copy has moved here:

https://github.com/usds/justice40-tool/tree/main/client/src/data/copy

It's organized by page. I was getting confused where piece of copy was. Now it's easier to find and easier to maintain. Also, we don't have to dig through components!

In this particular case, it was decided to place the version number on the methodology page. So the version number is defined here:

https://github.com/usds/justice40-tool/blob/1b17af84c824e1836f461788b8aa32b37e47df28/client/src/data/copy/methodology.tsx#L28